fieldservice_survey/wizard/fsm_order_survey_submit.py

```diff
@@ -0,0 +1,249 @@
+import logging
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
```

Comment:
```suggestion
from odoo import api, fields, models
```

get rid in favor of `self.env._`